pkg/cloud/cloud_test.go

```diff
@@ -71,3 +71,66 @@ func TestGetResources(t *testing.T) {
 		})
 	}
 }
+
+func TestGetBootstrapResources(t *testing.T) {
+	tc := []struct {
+		name     string
+		platform configv1.PlatformType
+		expected []client.Object
+	}{{
+		name:     "AWS resources returned as expected",
+		platform: configv1.AWSPlatformType,
+		expected: aws.GetBootstrapResources(),
+	}, {
+		name:     "OpenStack resources are empty, as the platform is not yet supported",
+		platform: configv1.OpenStackPlatformType,
+	}, {
+		name:     "GCP resources are empty, as the platform is not yet supported",
+		platform: configv1.GCPPlatformType,
+	}, {
+		name:     "Azure resources are empty, as the platform is not yet supported",
+		platform: configv1.AzurePlatformType,
+	}, {
```

Comment:
should we have an entry for azure stack hub as well?

Reply:
There is no platform for Azure Stack Hub :( We will have to implement additional logic to separate Azure and Azure Stack Hub similar to https://issues.redhat.com/browse/OCPCLOUD-1187 (well, starting from that, and then read the full infra status in operator as well to get it right).

Reply:
ack, thanks!